conf/documentation.conf

```diff
@@ -1204,6 +1204,13 @@ The algorithm used to calculate the VLAN in a VLAN pool.
 EOT
 guide_anchor=_introduction_to_role_based_access_control
 
+[advanced.active_directory_os_join_check_bypass]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Ability to disable the check if the os has been join to a domain.
```

Comment:
I find this confusing... we enable to disable the check?
Perhaps rephrase it like this:

Enable to bypass the operating system domain join verification.